mozci/task.py

```diff
@@ -125,7 +125,8 @@ class Task:
     label: Optional[str] = field(default=None)
     duration: Optional[int] = field(default=None)
     result: Optional[str] = field(default=None)
-    classification: Optional[str] = field(default=None)
+    state: Optional[str] = field(default=None)
```

Comment:
What do we use the state for?

Reply:
Nothing at the moment. I just figured it would be useful, e.g if someone were inspecting a task and noticed there was no result, they might think why? Then notice that the state says "running".

I can also imagine applications where we'd want to filter out running tasks, and saying `task.state == "completed"` is nicer than `task.result is not None`